core/common/src/main/java/alluxio/security/authorization/DefaultAccessControlList.java

```diff
@@ -60,27 +59,46 @@ public DefaultAccessControlList(AccessControlList acl) {
 
   /**
    * create a child file 's accessACL based on the default ACL.
+   * @param mode file's initial mode based on umask
```

Comment:
I'm not sure what an "initial" mode is. Doesn't this object already have a concept of "mode"?